doc/storage/api/api.rst

```diff
@@ -305,14 +332,195 @@ These definitions must be defined in the header file :file:`psa/internal_trusted
     .. retval:: PSA_ERROR_STORAGE_FAILURE
         The operation failed because the physical storage has failed (Fatal error).
 
-    Deletes the data from internal storage.
+   Introduced in Version 1.1. 
```

Comment:
`psa_its_remove()` has been in the API since 1.0. Lines 335-337 are not required here.